docs/conf.py

```diff
@@ -0,0 +1,37 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# For the full list of built-in configuration values, see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Project information -----------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
+
+import os
+import sys
+
+sys.path.insert(0,os.path.abspath('..'))
+
+project = 'HOI'
+copyright = '2023, Dishie'
+author = 'Dishie'
+release = '1.0.0'
+
+# -- General configuration ---------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#general-configuration
+
+extensions = ['sphinx.ext.todo','sphinx.ext.viewcode','sphinx.ext.autodoc']
```

Comment:
We will need more extensions later (gallery of examples, reference to scientific papers etc.). Here's the [complete list](https://github.com/brainets/frites/blob/master/docs/source/conf.py#L47)

Reply:
The 3 extensions `"sphinxcontrib.bibtex"`, `"sphinx_gallery.gen_gallery"` and `"sphinx.ext.linkcode"` gave error since they need certain types of files/ functions to work while, there are no such requirements currently.
So, I have commented them for the time being.